griptape/drivers/image_generation_model/amazon_bedrock_stable_diffusion_image_generation_model_driver.py

```diff
@@ -21,7 +21,7 @@ def text_to_image_request_parameters(
         image_height: int,
         negative_prompts: Optional[list[str]] = None,
         seed: Optional[int] = None,
-    ) -> dict:
+    ) -> Dict[str, Any]:
```

Comment:
Don't need to use `Typing.Dict` beyond python 3.7 (might be off there). Can replace with `dict[str, Any]`.